src/repositories/IngredientRepository.py

```diff
@@ -0,0 +1,68 @@
+import sqlite3
+from src.config.Configuration import Configuration
+from src.models.Ingredient import Ingredient
+from typing import List
+
+
+class IngredientRepository:
+
+    def __init__(self):
+        self.conf = Configuration.getInstance()
+        self.conn = sqlite3.connect(self.conf.db.file, check_same_thread=False)
+        pass
```

Comment:
remove the `pass`. aren't we using the `get_db` method in `db.py` instead of creating a new connection each time?